hadoop-tools/hadoop-aws/src/site/markdown/tools/hadoop-aws/testing.md

```diff
@@ -350,6 +350,17 @@ mvn verify -Dparallel-tests -Dprefetch -DtestsThreadCount=8
 mvn verify -Dparallel-tests -Dprefetch -Dscale -DtestsThreadCount=8
 ```
 
+## <a name="java17"></a> Java 17 Tests
+
+This module includes a test source tree which compiles and runs on
+Java 17+ _only_. This is to allow external libraries to be used
```

Comment:
Iceberg 1.8 requires Java 11+, instead of Java 17+

https://github.com/apache/iceberg/blob/9ce0fcf0af7becf25ad9fc996c3bad2afdcfd33d/.github/workflows/publish-snapshot.yml#L40

Reply:
really? Even better. I do seem to have to switch to a java17 JVM for a build though, but maybe that's because I do that for parquet.

This means I'll be able to wire yetus up to CI this even before we've got that java17 image up

Reply:
actually going to add a special fileformat profile which is needed to explicitly include the format stuff, rather than do it by java version.


why so? stops another loop getting into the build. there's enough going on there with avro and hbase being hadoop dependencies...keeping format (regression) testing optional will keep release engineers happy. Or at least not unhappy with me